KebPOSTests/ValidationTest.cs

```diff
@@ -0,0 +1,29 @@
+using KebPOS;
+
+namespace KebPOSTests;
+
+[TestFixture]
+public class ValidationTest
+{
+    [TestCase("13")]
+    [TestCase("1")]
+    [TestCase("17")]
+    [TestCase("300")]
+    public void IsValidIdInput_ShouldReturnTrue(string id)
+    {
+        var result = Validation.IsValidIdInput(id);
+
+        Assert.IsTrue(result);
+    }
+
```

Comment:
Could you please add more invalid cases: 
- a number that's too big for an int; 
- a string
- empty input